.vscode/launch.json

```diff
@@ -0,0 +1,44 @@
+// AUTOMATICALLY GENERATED FILE. PLEASE DO NOT MODIFY IT MANUALLY
```

Comment:
Please remove the files from the `.vscode` folder from your PR

Reply:
I don't want to have to do that every time -- is it OK if I remove it, commit, re-add it, then add it to a .gitignore?